tools/Google.Cloud.Tools.ReleaseManager/AddCommand.cs

```diff
@@ -123,5 +128,29 @@ protected override void ExecuteImpl(string[] args)
             Console.WriteLine($"Added {id} to the API catalog with the following configuration:");
             Console.WriteLine(api.Json.ToString(Formatting.Indented));
         }
+
+        private static Api.Service ParseServiceConfigYaml(string path)
+        {
+            if (path is null)
+            {
+                return null;
+            }
+
+            var deserializer = new Deserializer();
+            using (var reader = File.OpenText(path))
+            {
+                var yamlObject = deserializer.Deserialize(reader);
+                var serializer = new SerializerBuilder().JsonCompatible().Build();
+                var writer = new StringWriter();
```

Comment:
Should we use using block?

Reply:
No need for StringWriter - it's all in memory anyway. It wouldn't do any harm, but it wouldn't be useful either.